src/components/AppBar/ImportStudyList.tsx

```diff
@@ -13,9 +17,16 @@ import { ClassNameMap } from '@material-ui/core/styles/withStyles';
 import { PostAdd } from '@material-ui/icons';
 import React, { PureComponent } from 'react';
 
-import { openSnackbar } from '../../actions/AppStoreActions';
+import { addCustomEvent, openSnackbar } from '../../actions/AppStoreActions';
```

Comment:
Can you change these imports to use the `$lib` syntax? See https://github.com/icssc/AntAlmanac/blob/main/tsconfig.json and how some of the other files do these imports.

Reply:
I now use import { addCustomEvent, openSnackbar } from '$actions/AppStoreActions'; and the import line doesn't have errors.
But for lines 67, 89, 96, the error "Unsafe call of an `any` typed value." are shown.
Why is that?

Reply:
The line numbers must've changed because I'm looking at those lines and there aren't function calls on any of them. Can you send the contents of the lines that are getting errors? I also opened the project in my editor and the linter didn't see any errors so IDK how to reproduce the problem you're having.

Reply:
ok now it's changed. The error is 
`[plugin:vite:import-analysis] Failed to resolve import "$actions/AppStoreActions" from "src/components/AppBar/ImportStudyList.tsx". Does the file exist?
` happened in the file ImportStudyList.tsx.

Reply:
Use this updated [`vite.config.ts`](https://github.com/icssc/AntAlmanac/blob/main/apps/antalmanac/vite.config.ts). Your branch's version is missing all of the aliases for some reason.